windows_service/datadog_checks/windows_service/data/conf.yaml.example

```
@@ -36,6 +40,7 @@ instances:
       - <SERVICE_NAME_1>
       - name: <SERVICE_NAME_2>
       - startup_type: automatic
+        trigger_start: false
```

```suggestion
      - startup_type: automatic
        trigger_start: false
```

Are these two params used exclusively together? Or can either of them be a standalone param?

---

Hi @steveny91, `startup_type` and `trigger_start` can be used standalone they don't need to be used together.

---

Then shouldn't the config example have:
```suggestion
      - startup_type: automatic
      - trigger_start: false
```

Since they're standalone?

---

Hey @steveny91 , Yeah that makes sense. I've added this in my latest commit.